staker/staker.gno

```diff
@@ -128,8 +128,8 @@ func CollectReward(
 	tokenId uint64, // LP TokenID
 ) {
 	deposit, exist := deposits[tokenId]
-	require(exist, ufmt.Sprintf("[STAKER] staker.gno__CollectReward() || tokenId(%d) not staked", tokenId))
 
+	require(exist, ufmt.Sprintf("[STAKER] staker.gno__CollectReward() || tokenId(%d) not staked", tokenId))
```

Comment:
what's up with line changes ??

Reply:
> what's up with line changes ??

I ran the `gofumpt -s -w` commad in this file, so might be affected.